docs/exts/operators_and_hooks_ref.py

```diff
@@ -341,10 +204,24 @@ def render_content(self, *, tags: set[str] | None, header_separator: str = DEFAU
         raise NotImplementedError("Tou need to override render_content method.")
 
 
+def _common_render_list_content(*, header_separator: str, resource_type: str, template: str):
```

Comment:
@ferruzzi -> we talked about it recently. This is so much nicer now.

Reply:
Oh yeah, It's beautiful.  :heart_eyes: